src/server/cluster/cluster_family.cc

```diff
@@ -757,12 +770,20 @@ ClusterSlotMigration* ClusterFamily::AddMigration(std::string host_ip, uint16_t
       .get();
 }
 
-void ClusterFamily::RemoveFinishedIncomingMigrations() {
+void ClusterFamily::RemoveFinishedMigrations() {
```

Comment:
but now we should remove a migration from the map only if it is not in the config anymore, right?

Reply:
yes I will add this functionality later when remove finalization step